leakcanary-android-core/src/main/java/leakcanary/internal/activity/screen/LeakScreen.kt

```diff
@@ -127,7 +128,19 @@ internal class LeakScreen(
     spinner.setSelection(selectedLeakTraceIndex)
   }
 
-
+private fun parseLinks(str: String):String {
+    val words = str.split(" ")
+    var parsedString = ""
+    for (word in words) {
+      parsedString += if (Patterns.WEB_URL.matcher(word).matches()) {
```

Comment:
I wonder if we could use `Matcher.find()` here, but also this isn't a tight loop and splitting on spaces should work for all cases we care about. 👍